.github/workflows/ci.yaml

```diff
@@ -117,10 +117,17 @@ jobs:
         uses: taiki-e/install-action@v2
         with:
           tool: cargo-nextest
+      - name: "Install cargo insta"
+        uses: taiki-e/install-action@v2
+        with:
+          tool: cargo-insta
       - uses: Swatinem/rust-cache@v2
       - name: "Run tests"
         shell: bash
-        run: cargo nextest run --workspace --status-level skip --failure-output immediate-final --no-fail-fast -j 12
+        env:
+          NEXTEST_PROFILE: "ci"
+        run: cargo insta test --all-features --unreferenced reject --test-runner nextest
```

Comment:
Oh I didn't know about this, huh. Did you know about this @zanieb?

Reply:
Yeah I talked about this in my earlier pull request I linked :p I think the key here is setting the profile via environment variable? Nice work @MichaReiser